internal/errors/errors.go

```diff
@@ -0,0 +1,8 @@
+package errors
```

Comment:
What's the difference with this and https://github.com/openfga/openfga/tree/main/pkg/server/errors ?

Reply:
I want to nuke that file.  

No but seriously, I want to move the errors in that file to this one, but in the "sentinel error" form.

And then that file can have a big `switch` statement to do the `sentinel -> http status` conversion.

It's my vision for simplifying error handling, which is all over the place 😆